clickfile.py

```diff
@@ -472,6 +473,38 @@ def update_contracts(branch):
         update_npm=True,
     )
 
+    if with_uniswap:
+        update_contracts_from_git(
+            "https://github.com/neonlabsorg/Uniswap-V3-NEON.git",
+            "uniswap-v3",
+            branch="main",
+            update_npm=True,
+        )
+
+        pool_addr_path = (
```

Comment:
As I know there is only 1 countract where we need the replacement 
Do you want to check all here?

Reply:
It is a path to contract "PoolAddress.sol", we replace hash there only